Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-DM-1-A.yml

```diff
@@ -20,7 +20,7 @@ title:
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  You have an overview of all known security defects impacting particular applications.
+  You have an view on all known security defects impacting particular applications.
```

Comment:
should be either "an overview" or "a view" - I must say it's not obvious to me why "view" is more suitable?

Reply:
Good question, seemed more natural to me.